movo_common/movo_ros/src/movo/utils.py

```diff
@@ -112,7 +112,7 @@ def add_bytes(list_to_append,var2,bits):
     if bits % 2:
         return False
     
-    bytes_to_make = bits/8
+    bytes_to_make = bits//8
```

Comment:
pretty sneaky bug around integer division in python2 vs python3. keep this in mind when u guys are debugging stuff.
https://stackoverflow.com/questions/21316968/integer-division-in-python-2-and-python-3